platform/ext/target/nordic_nrf/nrf54l15dk_nrf54l15_cpuapp/partition/flash_layout.h

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2018-2022 Arm Limited. All rights reserved.
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __FLASH_LAYOUT_H__
+#define __FLASH_LAYOUT_H__
+
+/* Flash layout on NRF5340 Application MCU with BL2:
```

Comment:
ahem 5340 (multiple times)

Reply:
Yeah, that makes sense. I write it in the commit message that most of the files are plain copies of the 5340 files just to allow to build. These will have to be basically ported to the nRF54L15 when we want to add support for it upstream.